sklearn/titanic/README.md

```diff
@@ -0,0 +1,138 @@
+## Introduction
+
+This sample code illustrates how to start a *GCE* instance and train a model using scikit-learn on the instance. We will be using the [Titanic dataset](https://www.kaggle.com/c/titanic) from [Kaggle](https://www.kaggle.com/). The goal is to train a classifier to predict whether a given passenger on Titanic survived or not.
```

Comment:
You might want to use the full name Google Compute Engine for the first occurrence, and abbreviate it as just Compute Engine or GCE in subsequent mentions.

Reply:
Good point. Done.